searx/settings.yml

```diff
@@ -1740,6 +1740,12 @@ engines:
       require_api_key: false
       results: HTML
 
+  - name: moviepilot
+    shotcut: mp
+    engine: moviepilot
+    discovery_category: filme
```

Comment:
Oh, I called that `discover_type` in `moviepilot.py`, that's probably why CI might fail 🤦‍

Reply:
Yes, I have seen ;)

Reply:
Oh, that as well 🤦